src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -319,6 +320,11 @@ def map_keyvalue_to_featureflagvalue(keyvalue):
                     "key:{0} value:{1}\nFull exception: \n{2}".format(keyvalue.key, keyvalue.value, str(exception))
         raise ValueError(error_msg)
 
+    except (InvalidArgumentValueError, TypeError) as exception:
```

Comment:
Can we catch these in the same block as ValueError above? The cause is same for all cases and it should be okay to throw ValueError for all 3 error types.

Reply:
I think this is a good idea. This has been updated accordingly, thanks! Feature.py will catch and log `ValueError` only